internal/clients/http/ec2/ec2_client.go

```diff
@@ -169,7 +170,34 @@ func (c *ec2Client) ImportPubkey(ctx context.Context, key *models.Pubkey, tag st
 		return "", fmt.Errorf("cannot import SSH key %s: %w", key.Name, err)
 	}
 
-	return *output.KeyPairId, nil
+	return *output.KeyName, nil
+}
+
+func (c *ec2Client) GetPubkeyName(ctx context.Context, key *models.Pubkey) (string, error) {
```

Comment:
```suggestion
func (c *ec2Client) FindNameByFingerprint(ctx context.Context, key *models.Pubkey) (string, error) {
```

Perhaps? We should probably only pass fingerprint, not the whole pubkey as well.